src/main/resources/templates/jenkins/c/Jenkinsfile

```diff
@@ -26,50 +26,55 @@ pipeline {
+                            echo "--------------------info--------------------"
                             python3 --version
                             pip3 --version
                             gcc --version
                             # Generic debug infos:
                             # tree
-                            echo &quot;--------------------tests-------------------&quot;
+                            echo "--------------------info--------------------"
+                            echo "--------------------tests-------------------"
                             ls -la tests
-                            echo &quot;--------------------tests-------------------&quot;
-                            echo &quot;--------------------assignment--------------&quot;
+                            echo "--------------------tests-------------------"
+                            echo "--------------------assignment--------------"
                             ls -la assignment
-                            echo &quot;--------------------assignment--------------&quot;
+                            echo "--------------------assignment--------------"
 
                             cd tests
                             REQ_FILE=requirements.txt
-                            if [ -f &quot;$REQ_FILE&quot; ]; then
+                            if [ -f "$REQ_FILE" ]; then
                                 pip3 install --user -r requirements.txt
-                            else
-                                echo &quot;$REQ_FILE does not exist&quot;
+                            else 
+                                echo "$REQ_FILE does not exist"
                             fi
                             exit 0
                            '''
                         sh '''
                             #!/usr/bin/env bash
 
-                            shadowFilePath=&quot;../tests/testUtils/c/shadow_exec.c&quot;
+                            shadowFilePath="../tests/testUtils/c/shadow_exec.c"
 
-                            foundIncludeDirs=`grep -m 1 &apos;INCLUDEDIRS\s*=&apos; assignment/Makefile`
-                            echo &quot;Include: $foundIncludeDirs&quot;
+                            foundIncludeDirs=`grep -m 1 'INCLUDEDIRS\s*=' assignment/Makefile`
+                            echo "Include: $foundIncludeDirs"
 
-                            foundSource=`grep -m 1 &apos;SOURCE\s*=&apos; assignment/Makefile`
-                            foundSource=&quot;$foundSource $shadowFilePath&quot;
-                            echo &quot;Source: $foundSource&quot;
+
+                            foundSource=`grep -m 1 'SOURCE\s*=' assignment/Makefile`
+                            foundSource="$foundSource $shadowFilePath"
+                            echo "Source: $foundSource"
 
                             rm -f assignment/GNUmakefile
-                            rm -f assignment/Makefile
+                            rm -f assignment/makefile
```

Comment:
We write upper case `Makefile` everywhere else. (This might be an issue if the agents work with case sensitive paths?)